app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_inline => _("If you don't have admin permissions you can use slower filtered API."), :label => _('User level API') %>
```

Comment:
I think we are exposing too low-level stuff to the end-user. The checkbox should perhaps read `[x] Admin account` or something. The current name is too fuzzy and the inline help does not describe it enough. Why I should check it? What happens if I don't? I vote for *Administrator account* with help text of something like "Faster filtering for priviledged accounts".